Sources/Kitura/CodableRouter.swift

```diff
@@ -614,7 +627,7 @@ public struct CodableHelpers {
      * - Returns: An instance of `InputType` representing the decoded body data.
      */
     public static func readCodableOrSetResponseStatus<InputType: Codable>(_ inputCodableType: InputType.Type, from request: RouterRequest, response: RouterResponse) -> InputType? {
-        guard CodableHelpers.isContentTypeJSON(request) else {
+        guard CodableHelpers.isContentTypeJSON(request) || CodableHelpers.isContentTypeURLEncoded(request) else {
```

Comment:
This seems okay from a performance perspective: the guard should short-circuit evaluate in the existing case (where the content is JSON) and we will only evaluate the second clause in the case where the data is not JSON.  What would this look like if we add additional content types though?